addOns/pscanrulesAlpha/src/test/java/org/zaproxy/zap/extension/pscanrulesAlpha/SubResourceIntegrityAttributeScannerTest.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2019 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.pscanrulesAlpha;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+import org.parosproxy.paros.network.HttpMalformedHeaderException;
+import org.parosproxy.paros.network.HttpMessage;
+import org.zaproxy.zap.utils.ZapXmlConfiguration;
+
+public class SubResourceIntegrityAttributeScannerTest
+        extends PassiveScannerTest<SubResourceIntegrityAttributeScanner> {
+
+    @Test
+    public void shouldNotRaiseAlertGivenIntegrityAttributeIsPresentInLinkElement()
+            throws HttpMalformedHeaderException {
+        // Given
+        HttpMessage msg =
+                buildMessage(
+                        "<html><head><link rel=\"stylesheet\" href=\"https://site53.example.net/style.css\"\n"
```

Comment:
The `rel`/`crossorigin` attributes could be removed, they don't matter.

Reply:
I wanted to reuse the exact same example as in the specification

Reply:
What's the advantage though?

Reply:
The advantage is to use a well-known example.
It is a "reference" as it comes from a specification.
So people can understand it and find the origin.

I do agree about the fact that extra attributes are useless.

Reply:
OK, fine leaving as is then.